packages/hooks/use-fills.tsx

```diff
@@ -1,39 +1,73 @@
-import { useConnect } from "frontity";
+import { useConnect, warn } from "frontity";
 
-interface Fills {
+interface Fill {
+  slot: string;
+  library: string;
+  priority?: number;
+  props?: {
+    [key: string]: any;
+  };
+}
+
+interface FillsPackage {
   state: {
-    fills: {
-      [key: string]: {
-        slot: string;
-        library: string;
-        priority: number;
-        props: {
-          [key: string]: any;
-        };
-      };
-    };
+    fills: { [key: string]: Fill };
   };
   libraries: {
-    [key: string]: any;
+    fills: {
+      [key: string]: React.ComponentType;
+    };
   };
 }
 
-const useFills = (name) => {
-  const { state, libraries } = useConnect<Fills>();
-
-  Object.values(state.fills)
-    // Match only the fills for this name.
-    .find((fill) => fill.name === name)
-    // Sort fills by priority.
-    .sort((a, b) => a.priority > b.priority)
-    // Add real component to the array.
-    .map((fill) => {
-      const Fill = libraries.fills[fill.library];
-      return {
-        ...fill,
-        Fill,
-      };
-    });
+const useFills = (name: string) => {
+  const { state, libraries } = useConnect<FillsPackage>();
+
+  if (!name) {
+    warn("You should pass the name of the slot that you would like to fill!");
+    return [];
+  }
+
+  if (!Object.values(state.fills).find((slot) => slot.slot === name)) {
+    warn(`Could not find any slot with name ${name}. Is this what you want?`);
```

Comment:
This warning is not useful because the theme will declare many slots and most of them will be empty.